basket/news/tasks.py

```diff
@@ -256,10 +256,25 @@ def fxa_email_changed(data):
         # message older than our last update for this UID
         return
 
+    # Update SFDC
+    user_data = get_user_data(fxa_id=fxa_id, extra_fields=['id'])
+    if user_data:
+        sfdc.update(user_data, {'fxa_primary_email': email})
+    else:
+        # FxA record not found, try email
+        user_data = get_user_data(email=email, extra_fields=['id'])
+        if user_data:
+            sfdc.update(user_data, {'fxa_id': fxa_id})
+        else:
+            # No matching record for Email or FxA ID. Create one.
+            sfdc.add({'email': email, 'fxa_id': fxa_id)
```

Comment:
Similar question here: If we don't find the user with `email` or `fxa_id` we'll go ahead and create an entry. Do you want to  set `FxA_Primary_Email__c` in addition to `email`?
/cc @mozbrandon @andrewmozilla

Reply:
Yes, please set both. Thanks